extensions/VisualEditor/modules/ve-mw/dm/nodes/ve.dm.MWTransclusionNode.js

```diff
@@ -69,6 +69,25 @@ ve.dm.MWTransclusionNode.static.getHashObject = function ( dataElement ) {
 	};
 };
 
+/**
+ * return the type to be assigned to given data element. If this is a portable infobox, the proper
+ * node type was already defined and named.
+ *
+ * @param domElements array of clicked elements
+ * @param converter Converter object
+ * @returns {string} type of Node
+ */
+ve.dm.MWTransclusionNode.static.getDataElementType = function ( domElements, converter ) {
+	var isInline = this.isHybridInline( domElements, converter);
+
+	if ( isInline ) {
+		return ve.dm.MWTransclusionInlineNode.static.name;
+	}
+	return this.name === ve.dm.WikiaInfoboxTransclusionBlockNode.static.name ?
```

Comment:
true for the `ve.dm.WikiaInfoboxTransclusionBlockNode` but for normal templates, also on `dev` branch, `this.name` is: https://www.dropbox.com/s/luxanlkrlfmgwkk/Zrzut%20ekranu%202015-08-24%20o%2023.32.07.png
Will investigate it tomorrow as well, thanks!